resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -364,6 +439,108 @@
         performance when processing a BAM file in which
         variable-length data (read, qualities, tags) do not
         otherwise need to be decoded. Default is STRICT.
+    - name: trimming
+      label: Parameters for the trimming step
+      group:
+        - name: adapters
+          label: Adapter sequences
+          type: data:seq:nucleotide
+          required: false
+          description: |
+            Adapter sequence in FASTA format that will be removed from the read.
+            This field as well as 'Seed mismatches', 'Simple clip threshold' and 'Palindrome clip threshold'
+            parameters are  needed to perform Illuminacliping. 'Minimum adapter length' and 'Keep both reads' are
+            optional parameters.
+        - name: seed_mismatches
+          label: Seed mismatches
+          type: basic:integer
+          disabled: >
+            !trimming.adapters
+          required: false
+          description: |
+            Specifies the maximum mismatch count which will still allow a full match to be performed.
+            This field as well as 'Adapter sequence', 'Simple clip threshold' and 'Palindrome clip threshold'
+            parameters are needed to perform Illuminacliping.
+        - name: simple_clip_threshold
+          label: Simple clip threshold
+          type: basic:integer
+          disabled: >
+            !trimming.adapters
+          required: false
+          description: |
+            Specifies how accurate the match between any adapter etc. sequence must be against a read.
+            This field as well as 'Adapter sequences' and 'Seed mismatches' parameter are needed to perform Illuminacliping.
+        - name: min_adapter_length
```

Comment:
I'm just trying to understand the reasoning behind different parameters for trimming for paired and single-end workflows.

Reply:
Each uses different trimmomatic process, one is intended for single-end reads and the other for paired-end reads and some of the parameters are not used in the first one.